torchgeo/models/vit.py

```diff
@@ -727,7 +742,12 @@ def vit_huge_patch14_224(
             weights.get_state_dict(progress=True), strict=False
         )
         assert set(missing_keys) <= {'head.weight', 'head.bias'}
-        assert set(unexpected_keys) <= {'norm.weight', 'norm.bias'}
+        assert set(unexpected_keys) <= {
+            'norm.weight',
```

Comment:
Why did these change?

Reply:
Previously, we tested whether weights created with `features_only=True` could be loaded with `features_only=True` (and same for False).

Now, we test whether weights created with `features_only=False` can be loaded with `features_only=True` (or False).

I believe all of the pretrained weights we host were made with `features_only=False`, but can't be certain. If we want to be really sure, we could invert things to test True/False and False/True.

Reply:
Testing True/False and False/True actually revealed more bugs, let me make this change and fix.